app/adapters/branch.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+import ApplicationAdapter from 'travis/adapters/application';
+
+export default ApplicationAdapter.extend({
+
+  query(store, type, query) {
+    var repo_id = query['repository_id'];
+    delete query['repository_id'];
```

Comment:
This is more a stylistic thing, but note that you could use `query.repository_id` here, it’s more conventional.